charts/grafana/templates/_pod.tpl

```diff
@@ -27,6 +27,7 @@ initContainers:
     {{- end }}
     imagePullPolicy: {{ .Values.initChownData.image.pullPolicy }}
     securityContext:
+      runAsNonRoot: false
       runAsUser: 0
```

Comment:
Weird, wouldn't `runAsUser` handle this for you ? Or do you need both to enable root user.